docs/src/image_loading.md

```diff
@@ -0,0 +1,143 @@
+    <img src="images/image_loading_sequence.svg" alt="flash_config" width="100%">
+</p>
+
+The following outlines the steps carried out by the MCU RT during the SOC boot process:
+
+1. MCU ROM reads a SOC Configuration register (implementation specific) to determine the source of the images to load (Flash/BMC).
+2. Caliptra RT authorizes and loads Caliptra RT (refer to [Caliptra Subsystem boot flow](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#subsystem-boot-flow) for the detailed steps).
+3. Caliptra switches to Caliptra RT FW.
+4. Caliptra RT indicates to Recovery I/F that it is ready for the SOC manifest image (refer to [Caliptra Subsystem Recovery Sequence](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#caliptra-subsystem-recovery-sequence) for the detailed steps).
+5. Retrieve SOC Manifest
+
+   1. If image is coming from BMC, BMC FW transfers manifest to Recovery I/F
+   2. If Image is coming from Flash, MCU ROM transfers manifest from flash to Recovery I/F
+6. Caliptra RT transfers image to Caliptra Mailbox (MB) SRAM
+7. Caliptra RT will authenticate its image sitting in Caliptra MB SRAM
+8. Caliptra RT indicates to Recovery I/F that it is ready for the next image which should be the MCU RT Image (refer to [Caliptra Subsystem Recovery Sequence](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#caliptra-subsystem-recovery-sequence) for the detailed steps)..
+9. Retrieve MCU RT Image
+
+   1. If Image is coming from BMC, BMC FW transfers MCU RT Image to Recovery I/F
+   2. If Image is coming from Flash, MCU ROM transfers MCU RT Image to Recovery I/F
+10. Caliptra RT FW will read the recovery interface registers over AXI manager interface and write the image to MCU SRAM aperture
+11. Caliptra RT FW will instruct Caliptra HW to read MCU SRAM and generate the hash (Caliptra HW will use the SHA accelerator)
+12. Caliptra RT FW will use this hash and verify it against the hash in the SOC manifest.
+13. Once digest is verified, the EXEC/GO bit is set
+14. The EXEC/GO bit sets a Caliptra wire to MCU (as a consequence of setting the EXEC/GO bit in the previous step)
+15. MCU ROM detects Caliptra'go' wire to the MCU got set.
+16. MCU ROM passes a parameter using the FW HandOff table to indicate the image source (i.e. the image source where it booted from)
+17. MCU switches to MCU RT
+18. MCU RT retrieves the image source from HandOff table
+19. BMC or a similar platform component will now do MCTP enumeration flow to MCU over I3C.
+20. Retrieve SOC Images Metadata
+
+    1. If Image is coming from BMC, retrieve SOC Manifest and Image Metadata Collection through PLDM T5 flow
```

Comment:
SoC manifest has been retrieved in step(5).  We should start from image metadata collection via PLDM-T5 flow.

Reply:
Actually the Image Metadata Collection (IMC) is inside the SOC Manifest (https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/auth-manifest/README.md), and the PLDM package will send the SOC Manifest (with the IMC included). However, for the flash use case, since we know the structure of the SOC Manifest, we can get the IMC directly without having to load the preamble part. I can probably reword this to retrieve the Image Metadata Collection (as part of the SOC Manifest) through PLDM-T5 flow.